docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+---
+id: contributing
+title: Contributing to the Doc Hub
+slug: /contributing
+---
+
+The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to modify an existing document or create a new one.
+
+This page will walk you through how to edit an existing document or create a new one, how to display docs in the sidebar where desired, and how to make these changes in GitHub.
+
+Documents are written in Markdown (`.md`) files using [GitHub-flavored Markdown syntax](https://github.github.com/gfm/) and are compiled into static web pages. [MDX](https://mdxjs.com/) is also supported, meaning you may embed React components in content as needed. Refer to Docusaurus [Markdown features](https://v2.docusaurus.io/docs/markdown-features) for more technical information about these Markdown pages.
+
+<div className="flex">
+<div>
+<b>Notifications</b>
```

Comment:
Let's try and use as little HTML as possible in Markdown files (this can just be `**Notifications**`)

Reply:
Yep agreed, missed this